app/components/Settings/Settings.react.js

```diff
@@ -420,8 +424,13 @@ class Settings extends Component {
                                                     <strong><code>{connectorUrl}</code></strong>
                                                 </Link>
                                             </div>
+                                            <p>
+                                                {`Logged in as "${this.props.username}"`}
+                                                <br/>
+                                                <a onClick={this.props.logout}>Log Out</a>
```

Comment:
let's move this destructuring to the start of render where the rest of the `props` are destructured:
```
const {
   ....
   logout,
   ...
   username
} = this.props
```